File: Subprogramas/Parametros Variaveis/parametros_variaveis.lua
-- Lua
function printNumbers(...)
	-- Coloca todos os parâmetros em uma tabela
    local args = {...}  
    for i, number in ipairs(args) do
        print(number)
    end
end
-- Chama a função com parâmetros variáveis
printNumbers(1, 2, 3, 4, 5)  

File: Subprogramas/Parametros Variaveis/parametros_variaveis.py
# Python
def print_numbers(*numbers, **outros):
    for number in numbers:
        print(number)
    
    print()  
	
	# Acessa a chave e o valor do dicionário 'outros'
    for chave, valor in outros.items():  
        print(f"{chave}: {valor}")

# Chama a função com parâmetros variáveis
print_numbers(1, 2, 3, 4, 5, vermelho=10, verde=20, azul=30) 